messy/codegen/ipxact/pulp_open_pv_panel/pulp_open_core_component.xml

```diff
@@ -19,7 +18,7 @@ XMLSchema/IPXACT/1685-2022 http://www.accellera.org/XMLSchema/IPXACT/1685-2022/i
       <ipxact:componentInstantiation>
         <ipxact:name>tlms-interface</ipxact:name>
         <ipxact:language>system-c</ipxact:language>
-        <ipxact:libraryName>gap9lib</ipxact:libraryName>
+        <ipxact:libraryName>pulp_open_lib</ipxact:libraryName>
```

Comment:
Please keep the name consistent. I noticed that on all the other files the library name is `pulp_open`.